docs/content/dagster-cloud/account/managing-users.mdx

```diff
@@ -1,668 +1,90 @@
-src="/images/dagster-cloud/user-token-management/editor-permissions.png"
-width={1798}
-height={354}
-/>
-
-For Dagster Cloud Enterprise users, default user roles can be overridden at the [code location level](#code-location-permissions). For non-Enterprise users, the user will have the same level of access for all code locations in the deployment.
-
-**Note**: Having access to a deployment doesn't automatically grant access to Branch Deployments. [Branch deployment permissions](#branch-deployment-permissions) must be granted separately.
-
-#### Code location permissions
+## Adding users
 
-<Note>Code location permissions are a Dagster Cloud Enterprise feature.</Note>
+<Note>
+  <strong>Organization Admin</strong> or <strong>Admin</strong> permissions are
+  required to add users in Dagster Cloud.
+</Note>
 
-Dagster Cloud Enterprise users can override the deployment-level user role by setting permissions at the code location level. Overrides can only be performed if:
+Before you start, note that:
 
-- The user is not **Organization Admin**, and
-- The current user role for the deployment is **Viewer** or **Launcher**
+- **If using Google for SSO**, users must be added in Dagster Cloud before they can log in.
 
-For example, if a user is assigned the **Viewer** user role for a deployment, only a more permissive role can be assigned for a code location in the deployment. Specifically, an **Editor** or **Launcher** user role:
+- **If using a SAML-based solution like Okta**, users must be assigned to the Dagster app in the SSO portal to log in. By default, users will be granted Viewer permissions on each deployment. The default role can be adjusted by modifying the [`sso_default_role` deployment setting](/dagster-cloud/managing-deployments/deployment-settings-reference#sso-default-role).
 
-<!-- ![Overriding the Viewer user role for a code location](/images/dagster-cloud/user-token-management/code-location-override.png) -->
+1. Sign in to your Dagster Cloud account.
+2. Click the **user menu (your icon) > Cloud Settings**.
+3. Click the **Users** tab.
+4. Click **Add new user.**
+5. In the **User email** field, enter the user's email address.
+6. Click **Add user**. The user will be added to the list of users.
 
-<Image
-alt="Overriding the Viewer user role for a code location"
-src="/images/dagster-cloud/user-token-management/code-location-override.png"
-width={700}
-height={499}
-/>
+After the user is created, you can [add the user to teams and assign user roles for each deployment](#managing-users).
 
-If a user is an **Organization Admin** or a deployment-level **Admin** or **Editor**, they already have the most permissive user role possible for the code location. In this case, an override is unnecessary and the **Edit user role** button next to the code location will be disabled:
+---
 
-<!-- ![Override for a code location not allowed](/images/dagster-cloud/user-token-management/not-allowed-code-location-override.png) -->
+## Managing users
```

Comment:
'Managing' is a little vague here I think, there are a some actions that could fall in that category that don't require org admin permissions (i would call adding a user 'managing users'). Maybe we should list out the individual actions in their own sections?

Reply:
Adjusted this to `Managing user permissions` - more specific to the sections this guy contains, and I hope addresses your comment